pkg/resource/plugin/analyzer_plugin.go

```diff
@@ -311,6 +329,42 @@ func (a *analyzer) GetPluginInfo() (workspace.PluginInfo, error) {
 	}, nil
 }
 
+func (a *analyzer) Configure(policyConfig map[string]AnalyzerPolicyConfig) error {
+	label := fmt.Sprintf("%s.Configure(...)", a.label())
+	logging.V(7).Infof("%s executing", label)
+
+	c := make(map[string]*pulumirpc.PolicyConfig)
+
+	for k, v := range policyConfig {
+		var el pulumirpc.EnforcementLevel
+		if v.EnforcementLevel != "" {
+			el = marshalEnforcementLevel(v.EnforcementLevel)
+		}
+		c[k] = &pulumirpc.PolicyConfig{
+			EnforcementLevel: el,
+			Properties:       marshalMap(v.Properties),
+		}
+	}
+
+	_, err := a.client.Configure(a.ctx.Request(), &pulumirpc.ConfigureAnalyzerRequest{
+		PolicyConfig: c,
+	})
+	if err != nil {
+		rpcError := rpcerror.Convert(err)
+		// Handle the case where the policy pack doesn't implement a recent enough
+		// AnalyzerService to support the Configure method. Ignore the error as it
+		// just means the analyzer isn't capable of being configured in this way.
+		if rpcError.Code() == codes.Unimplemented {
```

Comment:
It seems like this would be an error case to me. I would think that we may want to return early if the `policyConfig` is null or len == 0. This should handle the case where the analyzer is older and does not support config. If the analyzer is older and we somehow have config, something has gone wrong... right?

Reply:
Right now, the implementation always creates initial config values based on the declared enforcement level for each policy... This happens even for older policy packs that don't support config. I'll need to somehow determine the policy pack doesn't support config (using one of the approaches I mention at https://github.com/pulumi/pulumi/pull/4015#discussion_r389030415), and if that's the case, avoid doing anything with config (i.e. don't send any config schema and don't call the `Configure` rpc).

Reply:
Fixed.